### ISSUE TITLE
Missing alt text for images

### ISSUE THREAD

**Issue by mohammad-suliman** [hasadna/datacity-businessgate#4]
## Problem description:
Several images on the website are missing alt text so an assistive technology like a screen reader can report them properly. This includes but not limited to:
* The first image that a screen reader meets (because it's the first element in the DOM from its perspective) witch is kind of a logo if I remember correctly
* Many image logos at the end of the page
* The icon for some person we came across while testing
* And so on...
## Solution:
* Give all none decorative images a none empty alt text describing meeningfully the image in a way that a screen reader user needs to hear
* Give the image an empty alt text in case it's decorative and a screen reader should ignore
